lib/src/Dscp/CLI/Common.hs

```diff
@@ -32,6 +32,10 @@ import Dscp.Resource.Network (NetCliParams (..), NetServParams (..))
 import Dscp.Web (NetworkAddress (..))
 import Paths_disciplina (version)
 
+import qualified Text.Parsec.String as Parsec
```

Comment:
This should not be a separate section, but rather united with a first one (project-independent).